cpp.md

```diff
@@ -258,6 +258,14 @@ do{
 //code 
 } while(condition); 
 ```
+### 4. Range based for loop
+```c
+double prices[5] = {4.99, 10.99, 6.87, 7.99, 8.49};
+for (double x : prices)
+cout << x << std::endl;  /*Here x initially represents the first member of the prices array.After displaying the
```

Comment:
code is self explanatory, please remove the comments

Reply:
comments removed !